### PR TITLE
Rating Screen

### PR DESCRIPTION
**Pull request by Nap1400**
image

### CODE REVIEW

**Comment by Nap1400** [comment]
image

File: mobile/src/screens/rating/Rating.type.ts [hunk 1]
```diff
@@ -1,8 +1,19 @@
-import { NativeStackScreenProps } from "@react-navigation/native-stack";
-import { RootStackParams } from "../../navigation";
-export type TRating = NativeStackScreenProps<RootStackParams, 'Rating'>
+import {StackScreenProps} from '@react-navigation/stack';
+import { ImageSourcePropType } from 'react-native';
+import { EAppNavigationRoutes, TAppNavigation, TAppNavigationRoutes } from '../../navigation';
+import { TRestaurant } from '../../types/data';
```
Comment (leo-he-1) [inline mobile/src/screens/rating/Rating.type.ts:4]: Were you using TRestaruant, EAppNavigationRoutes, TAppNavigationRoutes, and StackScreenProps for testing? Doesn't seem like they are being used anymore

File: mobile/src/screens/rating/Rating.view.tsx [hunk 2]
```diff
@@ -1,22 +1,58 @@
...
+    starRating,
+    textRating,
+    press,
 }) => {
     return (
-        <>
-            <Text>Hello World</Text>
-        </>
+        <Container
+            backgroundImage={require('../../assets/images/pizza_hut.png')}
```
Comment (leo-he-1) [inline mobile/src/screens/rating/Rating.view.tsx:30]: I think Lasha wants references like this to be an export. I did something similar in my pr https://github.com/Baby-Yoda-RN/food-hub/pull/53/files. Its in the images.ts file.

**Review by carizzuti — APPROVED** [review]

File: mobile/src/screens/rating/Rating.view.tsx [hunk 3]
```diff
@@ -1,22 +1,58 @@
 import React, {FC} from "react";
 import { Text,View,TouchableOpacity,Image } from "react-native";
```
Comment (leo-he-1) [inline mobile/src/screens/rating/Rating.view.tsx:2]: Looks like some of these imports are not being used.

**Review by leo-he-1 — APPROVED** [review]

**Review by Dulaya — CHANGES_REQUESTED** [review]
- I know you did not work on the Container component, but you have to change the ImageBackground in Container component to include imageStyle={imageStyle}. This will allows you to edit the borderRadius of the background image.
   - Make sure you edit the Container.type.ts also to include imageStyle.
- In Rating.view.tsx:
   - You have to add Header to Container and make sure the left icon navigate to previous screen.
   - The Pizza Hut icon needs a borderWidth and the green checkmark icon. 
   - You need to fix the paddings/margins through the screen. 
   - You need the green bullet point at Order Delivered. 
   - Make sure the Start Rating component works just like the Restaurant Review screen you did.
   - Button needs rounder borderRadius. 
- You have a lot of unused imports. 
Screenshot_1652130309

File: mobile/src/components/container/Container.tsx [hunk 4]
```diff
@@ -56,6 +57,7 @@ export const Container: FC<TContainer> = ({
         {backgroundColor: colors.background},
       ]}>
       <ImageBackground
+        imageStyle={imageStyle}
```
Comment (raynesmike) [inline mobile/src/components/container/Container.tsx:60]: Thanks for adding this style for the imageStyle for the container.

**Review by Josetf2396 — APPROVED** [review]

**Review by Dulaya — APPROVED** [review]

**Pull request merged**